### PR TITLE
Remove miniBoost

### LINKED ISSUE #33: miniBoost removal
couldn't mini boost just be removed from the source tree now ?

### PR DESCRIPTION
**Pull request by guparan**
Fixes #33.
Remove deprecated miniBoost dependency.

______________________________________________________

This PR: 
- [x] builds with SUCCESS for all platforms on the CI.
- [x] does not generate new warnings nor unit test failures.
- [x] does not break existing scenes.
- [x] does not break API compatibility.
- [x] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by damienmarchal** [comment]
Thank @guparan  for your PR.

**Comment by damienmarchal** [comment]
[ci-build]

**Comment by damienmarchal** [comment]
This pr looks ok, 
no one says stop in one week...
i rebuild it and if it succeed I merge it.

**Comment by damienmarchal** [comment]
[ci-build]

File: SofaKernel/CMakeLists.txt [hunk 1]
```diff
@@ -99,7 +99,7 @@ if(NOT WIN32)
 endif(NOT WIN32)
 
 
-## Try to find an installed boost. If boost is not found, use miniBoost.
+## Try to find an installed boost.
 find_package(Boost REQUIRED)
```
Comment (matthieu-nesme) [inline SofaKernel/CMakeLists.txt]: maybe a minimal version should be indicated here.

Comment (damienmarchal) [reply SofaKernel/CMakeLists.txt]: @guparan do I wait that to merge ?

Comment (guparan) [reply SofaKernel/CMakeLists.txt]: Any idea of what is the minimal version?

Comment (guparan) [reply SofaKernel/CMakeLists.txt]: Let's go for 1.54.0 as @fredroy pointed out concerning centOS build problems.

**Comment by damienmarchal** [comment]
Thanks @guparan for this PR.
Everything looks ok so I merge it.

**Pull request merged**